lib/Rx/Observable.php

```diff
@@ -1652,4 +1652,29 @@ public static function race(array $observables)
             return new RaceOperator();
         });
     }
+
+    /**
+     * Returns an Observable containing the value of a specified array index (if array) or property (if object) from
+     * all elements in the Observable sequence. If a property can't be resolved the observable will error.
+     *
+     * @param mixed $property
+     * @return Observable
+     *
+     * @demo pluck/pluck.php
+     * @operator
+     * @reactivex pluck
```

Comment:
Pluck is part of [map](http://reactivex.io/documentation/operators/map.html) on reactivex.io